test/test_klu_linear_solver_cache.jl

```diff
@@ -0,0 +1,95 @@
+using SparseArrays
```

Comment:
nitpick: by convention we put all the imports someplace central, for PowerFlows it's `runtests.jl`